rpm/util-linux.spec

```
@@ -198,6 +199,24 @@ Obsoletes: %{name}-docs
 %description doc
 Man and info pages for %{name}.
 
+%package systemd-fstrim
+Summary:   Systemd unit files for %{name}
+Group:     System
+Requires:  %{name} = %{version}-%{release}
+Requires(post): systemd
+
+%description systemd-fstrim
+Systemd unit files for fstrim
```

Description should end with full stop to make rpmlint happy:
```suggestion
Systemd unit files for fstrim.
```